meson.build

```diff
@@ -110,6 +110,10 @@ client_protos = declare_dependency(
 conf_data = configuration_data()
 conf_data.set10('HAVE_GDK_PIXBUF', gdk_pixbuf.found())
 
+if cc.has_function('explicit_bzero')
+	conf_data.set('HAVE_EXPLICIT_BZERO', 1)
```

Comment:
Can we use `set01` here to avoid the "forgot to include conf data header" issue? (Also need to replace `#ifdef` with `#if`)

Reply:
You mean set10?

Reply:
Yeah, sorry

Reply:
done.